app/models/rating/strictness.rb

```diff
@@ -45,14 +45,13 @@ def strictness_per_reviewer
 
   def ratings
     @ratings ||= Rating
-                  .joins('JOIN applications ON ratings.rateable_id = applications.id')
-                  .where(rateable_type: 'Application')
+                  .includes(:application)
```

Comment:
Not sure this is an equivalent refactor. `joins` is an INNER and `includes` is an OUTER JOIN if I recall correctly, meaning you will project values don't have matching association.

Reply:
Yep, you're totally right!

Should be
```ruby
Rating.joins(applications)
```

Will do the correct query then:
```ruby
> Rating.joins(:application).where('applications.season_id' => season.id)
  Rating Load (5.4ms)  SELECT  "ratings".* FROM "ratings" INNER JOIN "applications" ON "applications"."id" = "ratings"."application_id" WHERE "applications"."season_id" = $1 LIMIT $2  [["season_id", 1], ["LIMIT", 11]]
```